pkg/query-service/app/traces/v3/query_builder.go

```diff
@@ -235,22 +235,37 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, tableName str
 		having = " having " + having
 	}
 
-	// Select the aggregate value for interval
-	queryTmpl :=
-		"SELECT toStartOfInterval(timestamp, INTERVAL %d SECOND) AS ts" + selectLabels +
-			", %s as value " +
-			"from " + constants.SIGNOZ_TRACE_DBNAME + "." + constants.SIGNOZ_SPAN_INDEX_TABLENAME +
-			" where " + spanIndexTableTimeFilter + "%s " +
-			"group by %s%s " +
-			"order by %s"
+	var queryTmpl string
+
+	if panelType == v3.PanelTypeTable {
+		queryTmpl =
+			"SELECT" + selectLabels +
+				" %s as value " +
+				"from " + constants.SIGNOZ_TRACE_DBNAME + "." + constants.SIGNOZ_SPAN_INDEX_TABLENAME +
+				" where " + spanIndexTableTimeFilter + "%s" +
+				"%s%s " +
+				"order by %s"
```

Comment:
this will produce an invalid query when orderBy is empty.

Reply:
Good catch, fixed.
Thanks